web.esphome.io/src/components/ew-header-menu.ts

```diff
@@ -54,7 +54,6 @@ export class EWHeaderMenu extends LitElement {
 
   static styles = css`
     mwc-button {
-      --mdc-theme-primary: black;
```

Comment:
You're changing this button to be a different color. Why?

Reply:
In order for both light and dark to work, I've removed the override of the theme in each component, if I don't remove this the button text stays black in dark mode...
Without removing:
<img width="177" alt="image" src="https://github.com/esphome/dashboard/assets/771817/0ff9b618-628a-4bdb-ad87-9a80fa0cb3fe">
Removed:
<img width="188" alt="image" src="https://github.com/esphome/dashboard/assets/771817/0132922f-96fc-4264-ba9b-eeb3f06e4874">